config.lua

```diff
@@ -42,7 +42,14 @@ Config.Zones = {
 		Type  = -1
 	},
 
-	GiveBackVehicle = {},						--Disable Vehicle Rent
+	GiveBackVehicle = {
```

Comment:
Remove the whole key, value if this is not going to be used since its disabled in the menu.